tests/test_models.py

```diff
@@ -7,6 +7,16 @@
 
 yml = """
 auth:
+  teams:
+    admins:
+      - your_team
+    readers:
+      - sibling_team
+      - - other_team_1
+        - other_team_2
```

Comment:
It seems to work but is it intended? I am not sure if this "hierarchy" makes sense here.

Reply:
This is not hierarchy per se, this is workaround for the cases like:
```yaml
READER_TEAMS:
  - sibling_team
  - @@@./teams/teams_set_2.yaml #in file - [other_team_1, other team_2]
```
then the result would be nested:
```
- sibling team
- - other_team_1
- - other_team_2
```
however effectively we want tit flatterned.